packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -135,7 +137,7 @@ export class Snapshot {
         );
 
         if (isNotDone) {
-          throw new Error('Snapshot is still being processed');
+          throw new Error(SnapshotErrors.operationTimedOut);
```

Comment:
I think exponentialBackoff here does not fit properly our needs compared to async-retry, I'll get you a suggestion built up 🔜:tm: